### ISSUE TITLE
Visual Mode: InputDragger component tweaks

### ISSUE THREAD

**Issue by JonnyBurger** [remotion-dev/remotion#6759]
Current experience:

https://github.com/user-attachments/assets/1ee63478-9db1-4dc3-88dc-f6c0475a5a48

1. The padding when the input is focused is way too big.
2. We might not need an outline while dragging. This is for keyboard navigation, but you cannot drag with the keyboard.
3. When tabbing with the keyboard, it should actually focus the input field right away, because that is the only interaction that can be done.
4. We don't need the underline.
5. The text should lighten up slightly to indicate a hover.
6. The font size can become smaller, it doesn't align with the left label.

Compare this to After Effects Experience:

https://github.com/user-attachments/assets/94458827-d9d1-4a78-bb90-8c640d0b0c91
